libs/server/Resp/CmdStrings.cs

```diff
@@ -123,6 +123,8 @@ public static ReadOnlySpan<byte> GetConfig(ReadOnlySpan<byte> key)
         public static ReadOnlySpan<byte> RESP_ERR_GENERIC_REGISTERCS_UNSUPPORTED_CLASS => "ERR unable to register one or more unsupported classes."u8;
         public static ReadOnlySpan<byte> RESP_ERR_GENERIC_VALUE_IS_NOT_INTEGER => "ERR value is not an integer or out of range."u8;
         public static ReadOnlySpan<byte> RESP_ERR_GENERIC_UKNOWN_SUBCOMMAND => "ERR Unknown subcommand. Try LATENCY HELP."u8;
+        public static ReadOnlySpan<byte> RESP_ERR_GENERIC_OVERFLOW_ERROR => "increment or decrement would overflow"u8;
```

Comment:
>RESP_ERR_GENERIC_OVERFLOW_ERROR [](http://example.com/codeflow?start=41&length=31)

Not being used?

Reply:
Removed, since I am only addressing invalid type error